Frameworks/SafariServices/SFSafariWebViewController.mm

```diff
@@ -177,4 +207,8 @@ - (void)_cancel {
     [self dismissViewControllerAnimated:NO completion:nil];
 }
 
+- (void)dealloc {
+    _webView.delegate = nil;
```

Comment:
No, if it's null, this will be a no-op.

---
In reply to: [83332466](https://github.com/Microsoft/WinObjC/pull/1147#discussion_r83332466) [](ancestors = 83332466)